.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Install Betamax certificate
+        run: sudo keytool -importcert -keystore $JAVA_HOME/jre/lib/security/cacerts -file betamax.pem -storepass changeit -noprompt
+
+      - name: Cache Gradle packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('build.gradle') }}
+          restore-keys: ${{ runner.os }}-gradle
+
+      - name: Build with Gradle
+        run: ./gradlew build
```

Comment:
In our readme we only mention `gradlew assemble`/`gradlew check` so I had to look this up. Noting here that `build` seems to be a combination of the two, according to the image below https://docs.gradle.org/3.3/userguide/java_plugin.html#sec:java_tasks :)

Reply:
Hehe yes, TIL too. I was thinking that I should maybe change the `README`, but I kind of like the two explicit steps there but for CI it made sense with a single step.